src/utils/api/poloniexApi.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @module utils/poloniexApi
+ * @desc Api resources for Poloniex
+ */
+
+ const BASE_URL = 'https://poloniex.com';
+
+ // TODO: Refactor to be a pure api client with just request and response operations; e.g., remove time formatting here by passing as prop in params.
+ /**
+  * @function getData TODO: change name, is a bit generic if we think of this client as globally reuseable and if it's not part of a class there could also be collision imports
+  * @param {object} params
+  * @description Fetches price history data from Poloniex API and invokes setState with new data.
+  */
+async function getData(params, currCrypto) {
+  // variables required to fetch price data
+  const { symbol } = this.state.currCrypto; // FIXME: this.state does not exist here. currCrypto is a new param, find all uses of 'getData' and pass in currCrypto or just include in params obj
+  const { start, end, period, timeRange } = params;
+
+  // format start and end dates to unix timestamp for API request
+  const startUnix = start.unix();
+  const endUnix = end.unix();
+
+  // TODO: remove this time formatting from this api method
+  // format start and end dates to domString for HTML5 date input
+  const startString = start.format('YYYY-MM-DD');
+  const endString = end.format('YYYY-MM-DD');
```

Comment:
called out in the todo but this signals a slight violation of separation of concerns, I wouldn't expect async api call to do this much with date formatting and would lean on having a clean set of params passed in or use a serializer/transformer (see Axios docs for transformers).

Reply:
I'll look into that, thanks.